nuheat/util.py

```diff
@@ -66,3 +66,29 @@ def nuheat_to_celsius(nuheat_temperature):
     """
     fahrenheit = nuheat_to_fahrenheit(nuheat_temperature)
     return fahrenheit_to_celsius(fahrenheit)
+
+def get_api_url(config, brand="NUHEAT"):
```

Comment:
rather than passing `config` around as arguments, why not simply `import` it in this file?